### PR TITLE
Ana/fix my own mess in connection.js

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

It seems that I had it all wrong (as you can tell by the comment I left 😅 )

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: tests/unit/specs/store/connection.spec.js [hunk 1]
```diff
@@ -86,24 +86,23 @@ describe(`Module: Connection`, () => {
     localStorage.clear()
   })
 
-  it(`assigns the user the fallback network if there is no persisted network 
+  it.only(`assigns the user the fallback network if there is no persisted network
```
Comment (faboweb) [inline tests/unit/specs/store/connection.spec.js:89]: attention

Comment (faboweb) [reply tests/unit/specs/store/connection.spec.js:89]:
```suggestion
  it(`assigns the user the fallback network if there is no persisted network 
```

Comment (Bitcoinera) [reply tests/unit/specs/store/connection.spec.js]: oops! Thank you. I always forget those 🤦‍♀

**Comment by Bitcoinera** [comment]
This one is good to go!

**Pull request merged**